pkg/bench/rttanalysis/testdata/benchmark_expectations

```diff
@@ -27,7 +27,7 @@ exp,benchmark
 7,AlterTableUnsplit/alter_table_unsplit_at_1_value
 9,AlterTableUnsplit/alter_table_unsplit_at_2_values
 11,AlterTableUnsplit/alter_table_unsplit_at_3_values
-5,Audit/select_from_an_audit_table
+3,Audit/select_from_an_audit_table
```

Comment:
Nit: Could you also add a test without doing audit. Just to make sure the round trips are the same.

Reply:
Added a new test here to confirm they are the same.